benchmarks.ts

```diff
@@ -46,6 +49,58 @@ function measure<T=void>(message: (t:T) => string, callback: () => T, minMillise
   return result;
 }
 
+function intersectBySorting(
```

Comment:
Add comment ``/** calls `callback` for each key and pair of values that is in both `tree1` and `tree2` (O(n)) */``